index.js

```diff
@@ -1,18 +1,58 @@
 /***** Deliverable 1 *****/
 const header = document.querySelector("#header")
-console.log("Here's your header:", header)
+// console.log("Here's your header:", header)
 
 
 /***** Deliverable 2 *****/
-
+let headerElement = document.querySelector('#header')
+headerElement.style.color = 'blue'
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+
+// update image
+let imgElement = document.querySelector('img')
+imgElement.src = traveler.photo
+imgElement.alt = 'raffy picture'
 
+// update name
+let nameElement = document.querySelector('h2')
+nameElement.textContent = traveler.name
+
+// update nickname
+let nicknameElement = document.querySelector('em')
+nicknameElement.textContent = traveler.nickname
 
 /***** Deliverable 4 *****/
+let parent = document.querySelector('ul')
+
+traveler.animalSightings.forEach(function(animalSighting) {
+
+    // create parent li
+    let li = document.createElement('li')
+    li.setAttribute('data-id', animalSighting.id)
```

Comment:
For dataset attributes, you can also use this syntax:

```js
li.dataset.id = sighting.id
```

This will add a `data-id` attribute to the element, like this:

```html
<li data-id="1">...</li>
```